### PR TITLE
merge is not a function

### PR DESCRIPTION
**Pull request by makuzaverite**
### Since webpack merge package has been updated to use destructure

Doc example here https://webpack.js.org/guides/production/#setup

was giving the following error 

image

Solution was simply to edit and add destructure to merge package imports like this
solution

Hope it Helps!

### CODE REVIEW

**Comment by makuzaverite** [comment]
Where @EugeneHlushko

**Comment by makuzaverite** [comment]
Do you mean file i've changed where be specific

**Comment by anshumanv** [comment]
> @evilebottnawi Do you mean file i've changed where be specific?

See your diff, you've changed `__` to `**` everywhere, revert those changes

File: src/content/guides/production.md [hunk 1]
```diff
@@ -27,22 +27,21 @@ In this guide, we'll dive into some of the best practices and utilities for buil
 
 T> This walkthrough stems from [Tree Shaking](/guides/tree-shaking) and [Development](/guides/development). Please ensure you are familiar with the concepts/setup introduced in those guides before continuing on.
 
-
 ## Setup
 
-The goals of _development_ and _production_ builds differ greatly. In _development_, we want strong source mapping and a localhost server with live reloading or hot module replacement. In _production_, our goals shift to a focus on minified bundles, lighter weight source maps, and optimized assets to improve load time. With this logical separation at hand, we typically recommend writing __separate webpack configurations__ for each environment.
+The goals of _development_ and _production_ builds differ greatly. In _development_, we want strong source mapping and a localhost server with live reloading or hot module replacement. In _production_, our goals shift to a focus on minified bundles, lighter weight source maps, and optimized assets to improve load time. With this logical separation at hand, we typically recommend writing **separate webpack configurations** for each environment.
```
Comment (snitin315) [inline src/content/guides/production.md:32]: here

File: src/content/guides/production.md [hunk 2]
```diff
@@ -27,22 +27,21 @@ In this guide, we'll dive into some of the best practices and utilities for buil
...
 
 Let's start by installing `webpack-merge` and splitting out the bits we've already worked on in previous guides:
 
-``` bash
+```bash
 npm install --save-dev webpack-merge
 ```
 
-__project__
+**project**
```
Comment (snitin315) [inline src/content/guides/production.md:42]: here

File: src/content/guides/production.md [hunk 3]
```diff
@@ -56,9 +55,9 @@ __project__
   |- /node_modules
 ```
 
-__webpack.common.js__
+**webpack.common.js**
```
Comment (snitin315) [inline src/content/guides/production.md:59]: here

File: src/content/guides/production.md [hunk 4]
```diff
@@ -81,10 +80,10 @@ __webpack.common.js__
 + };
 ```
 
-__webpack.dev.js__
+**webpack.dev.js**
```
Comment (snitin315) [inline src/content/guides/production.md:83]: here

File: src/content/guides/production.md [hunk 5]
```diff
@@ -96,10 +95,10 @@ __webpack.dev.js__
 + });
 ```
 
-__webpack.prod.js__
+**webpack.prod.js**
```
Comment (snitin315) [inline src/content/guides/production.md:98]: here

File: src/content/guides/production.md [hunk 6]
```diff
@@ -111,14 +110,13 @@ In `webpack.common.js`, we now have setup our `entry` and `output` configuration
 
 Note the use of `merge()` calls in the environment-specific configurations to include our common configuration in `webpack.dev.js` and `webpack.prod.js`. The `webpack-merge` tool offers a variety of advanced features for merging but for our use case we won't need any of that.
 
-
 ## NPM Scripts
 
 Now, let's modify our npm scripts to use the new configuration files. For the `start` script, which runs `webpack-dev-server`, we will use `webpack.dev.js`, and for the `build` script, which runs `webpack` to create a production build, we will use `webpack.prod.js`:
 
-__package.json__
+**package.json**
```
Comment (snitin315) [inline src/content/guides/production.md:117]: here

File: src/content/guides/production.md [hunk 7]
```diff
@@ -156,24 +154,24 @@ Feel free to run those scripts and see how the output changes as we continue add
 
 Many libraries will key off the `process.env.NODE_ENV` variable to determine what should be included in the library. For example, when `process.env.NODE_ENV` is not set to `'production'` some libraries may add additional logging and testing to make debugging easier. However, with `process.env.NODE_ENV` set to `'production'` they might drop or add significant portions of code to optimize how things run for your actual users. Since webpack v4, specifying [`mode`](/configuration/mode/) automatically configures [`DefinePlugin`](/plugins/define-plugin) for you:
 
-__webpack.prod.js__
+**webpack.prod.js**
```
Comment (snitin315) [inline src/content/guides/production.md:157]: here

File: src/content/guides/production.md [hunk 8]
```diff
@@ -156,24 +154,24 @@ Feel free to run those scripts and see how the output changes as we continue add
...
     mode: 'production',
   });
 ```
 
-T> Technically, `NODE_ENV` is a system environment variable that Node.js exposes into running scripts. It is used by convention to determine dev-vs-prod behavior by server tools, build scripts, and client-side libraries. Contrary to expectations, `process.env.NODE_ENV` is not set to `'production'` __within__ the build script `webpack.config.js`, see [#2537](https://github.com/webpack/webpack/issues/2537). Thus, conditionals like `process.env.NODE_ENV === 'production' ? '[name].[hash].bundle.js' : '[name].bundle.js'` within webpack configurations do not work as expected.
+T> Technically, `NODE_ENV` is a system environment variable that Node.js exposes into running scripts. It is used by convention to determine dev-vs-prod behavior by server tools, build scripts, and client-side libraries. Contrary to expectations, `process.env.NODE_ENV` is not set to `'production'` **within** the build script `webpack.config.js`, see [#2537](https://github.com/webpack/webpack/issues/2537). Thus, conditionals like `process.env.NODE_ENV === 'production' ? '[name].[hash].bundle.js' : '[name].bundle.js'` within webpack configurations do not work as expected.
```
Comment (snitin315) [inline src/content/guides/production.md:168]: here

File: src/content/guides/production.md [hunk 9]
```diff
@@ -156,24 +154,24 @@ Feel free to run those scripts and see how the output changes as we continue add
...
 
 If you're using a library like [`react`](https://reactjs.org/), you should actually see a significant drop in bundle size after adding `DefinePlugin`. Also, note that any of our local `/src` code can key off of this as well, so the following check would be valid:
 
-__src/index.js__
+**src/index.js**
```
Comment (snitin315) [inline src/content/guides/production.md:172]: here

File: src/content/guides/production.md [hunk 10]
```diff
@@ -206,15 +203,14 @@ Note that while the [`TerserPlugin`](/plugins/terser-webpack-plugin/) is a great
 
 If you decide to try another minification plugin, just make sure your new choice also drops dead code as described in the [tree shaking](/guides/tree-shaking) guide and provide it as the [`optimization.minimizer`](/configuration/optimization/#optimizationminimizer).
 
-
 ## Source Mapping
 
 We encourage you to have source maps enabled in production, as they are useful for debugging as well as running benchmark tests. That said, you should choose one with a fairly quick build speed that's recommended for production use (see [`devtool`](/configuration/devtool)). For this guide, we'll use the `source-map` option in the _production_ as opposed to the `inline-source-map` we used in the _development_:
 
-__webpack.prod.js__
+**webpack.prod.js**
```
Comment (snitin315) [inline src/content/guides/production.md:210]: and here

File: src/content/guides/production.md (hunk 1 above)
Comment (EugeneHlushko) [inline src/content/guides/production.md:32]: Like so, and other occurances
```suggestion
The goals of _development_ and _production_ builds differ greatly. In _development_, we want strong source mapping and a localhost server with live reloading or hot module replacement. In _production_, our goals shift to a focus on minified bundles, lighter weight source maps, and optimized assets to improve load time. With this logical separation at hand, we typically recommend writing __separate webpack configurations__ for each environment.
```

File: src/content/guides/production.md (hunk 2 above)
Comment (makuzaverite) [reply src/content/guides/production.md:42]: You mean i have to use (__) instead of (**)

File: src/content/guides/production.md (hunk 10 above)
Comment (makuzaverite) [reply src/content/guides/production.md:210]: you mean i have to change (**) to (__)

**Comment by makuzaverite** [comment]
Fixed Thank you for you time it was my prettier setup but i have fixed it I genuely appreciate all of you

File: src/content/guides/production.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Production
-sort: 17
+sort: 18
```
Comment (montogeek) [inline src/content/guides/production.md:3]: Why this change it the order?

File: src/content/guides/production.md [hunk 12]
```diff
@@ -36,13 +36,13 @@ While we will separate the _production_ and _development_ specific bits out, not
 
 Let's start by installing `webpack-merge` and splitting out the bits we've already worked on in previous guides:
 
-``` bash
+```bash
```
Comment (montogeek) [inline src/content/guides/production.md:39]: Please revert all these changes.

File: src/content/guides/production.md (hunk 11 above)
Comment (makuzaverite) [reply src/content/guides/production.md:3]: i added my self my github username

**Comment by EugeneHlushko** [comment]
@makuzaverite can you sign the CLA please?

File: src/content/guides/production.md [hunk 13]
```diff
@@ -1,9 +1,10 @@
 ---
 title: Production
-sort: 17
+sort: 18
```
Comment (EugeneHlushko) [inline src/content/guides/production.md:3]: reverting to 17
```suggestion
sort: 17
```

**Comment by EugeneHlushko** [comment]
Thanks!
